components/admin_console/billing/billing_history.tsx

```diff
@@ -117,82 +116,108 @@ const getPaymentStatus = (status: string) => {
-                            style='currency'
-                            currency='USD'
+                    </th>
+                    <th className='BillingHistory__table-headerTotal'>
+                        <FormattedMessage
+                            id='admin.billing.history.total'
+                            defaultMessage='Total'
                         />
-                    </td>
-                    <td>
-                        {getPaymentStatus(info.status)}
-                    </td>
-                    <td className='BillingHistory__table-invoice'>
-                        {info.invoice &&
-                            <a href={info.invoice}>
-                                <i className='icon icon-file-pdf-outline'/>
-                            </a>
-                        }
-                    </td>
+                    </th>
+                    <th>
+                        <FormattedMessage
+                            id='admin.billing.history.status'
+                            defaultMessage='Status'
+                        />
+                    </th>
+                    <th>{''}</th>
                 </tr>
-            ))}
-        </table>
+                {billingHistory.map((info: any) => (
+                    <tr
+                        className='BillingHistory__table-row'
+                        key={info.id}
+                    >
+                        <td>
+                            <FormattedDate
+                                value={info.date}
+                                month='2-digit'
+                                day='2-digit'
+                                year='numeric'
+                            />
+                        </td>
+                        <td>
+                            <div>{info.product_name}</div>
+                            <div className='BillingHistory__table-bottomDesc'>{info.charge_desc}</div>
+                        </td>
+                        <td className='BillingHistory__table-total'>
+                            <FormattedNumber
+                                value={info.total}
+                                // eslint-disable-next-line react/style-prop-object
+                                style='currency'
+                                currency='USD'
+                            />
+                        </td>
+                        <td>
+                            {getPaymentStatus(info.status)}
+                        </td>
+                        <td className='BillingHistory__table-invoice'>
+                            {info.invoice &&
+                                <a href={info.invoice}>
+                                    <i className='icon icon-file-pdf-outline'/>
+                                </a>
+                            }
+                        </td>
+                    </tr>
+                ))}
+            </table>
+            {paging}
+        </>
```

Comment:
This is the only change to the section above, Github is not great with the indentation changes